plugins/c8y_configuration_plugin/src/smartrest.rs

```diff
@@ -0,0 +1,30 @@
+use c8y_smartrest::error::SmartRestSerializerError;
+use c8y_smartrest::smartrest_serializer::SmartRest;
+use c8y_smartrest::topic::C8yTopic;
+use mqtt_channel::Message;
+
+pub trait GetSmartRestMessage {
```

Comment:
This trait can even be moved to the `c8y_smartrest` module as it can be reused by all operation plugins.